docs/installation-configuration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: MSSQL Setup
```

Comment:
All over the Pulumi Registry, the title is standardized on `<provider> Installation & Configuration`. Please update.